lms/static/js/student_account/views/LoginView.js

```diff
@@ -102,6 +102,14 @@
             },
 
             saveError: function( error ) {
+                var url;
+                if(error.status === 418) {
+                    url = error.responseText;
+                    if(url) {
+                        window.location = url;
+                        return;
+                    }
+                }
```

Comment:
this would be cleaner, I think:

```javascript
var url = error.responseText;
if(url && error.status === 418) {
    window.location = url;
    return;
}
```